.github/readme.md

```diff
@@ -1,11 +1,64 @@
-Drone is a Continuous Delivery system built on container technology. Drone uses a simple YAML configuration file, a superset of docker-compose, to define and execute Pipelines inside Docker containers. 
+# [Drone](https://www.drone.io/) <img src="https://github.com/drone/brand/blob/master/screenshots/screenshot_build_success.png" style="max-width:100px;" />
+**Welcome to the Drone codebase, we are thrilled to have you here!** 
 
-<br/>
 
-<img src="https://github.com/drone/brand/blob/master/screenshots/screenshot_build_success.png" style="max-width:100px;" />
+## What is Drone?
+Drone is a continuous delivery system built on container technology. Drone uses a simple YAML configuration file, a superset of docker-compose, to define and execute Pipelines inside Docker containers. we have two versions available: the [Enterprise Edition](https://github.com/drone/drone/blob/master/BUILDING) and the [Community Edition](https://github.com/drone/drone/blob/master/BUILDING_OSS)
+
+_Please note the official Docker images run the Drone Enterprise distribution. If you would like to run the Community Edition you can build from source by following the instructions in [BUILDING_OSS](https://github.com/drone/drone/blob/master/BUILDING_OSS)._
+ 
+## Table of Contents
+
+- [What is Drone?](#what-is-drone)
+- [Table of Contents](#table-of-contents)
+- [Community](#community)
+- [Contributing](#contributing)
+- [Code of Conduct](#code-of-conduct)
+- [Core Team](#core-team)
+- [Setup Documentation](#setup-documentation)
+- [Usage Documentation](#usage-documentation)
+- [Sample Pipeline Configuration](#sample-pipeline-configuration)
+- [Plugin Index](#plugin-index)
+
+## Community
+---
```

Comment:
the `---` can be removed since these are meant to be page breaks.  Also github already adds an underline to `h2` headers via their css style sheet.